### ISSUE TITLE
[rx-51] wrong keyboard layout in X

### ISSUE THREAD

**Issue by pavelmachek** [postmarketOS/pmbootstrap#733]
It works ok in Debian. Xorg.0.log shows:

```
[    48.847] (**) evdev: TWL4030 Keypad: Device: "/dev/input/event1"
[    48.847] (--) evdev: TWL4030 Keypad: Vendor 0x1 Product 0x1
[    48.848] (--) evdev: TWL4030 Keypad: Found keys
[    48.848] (II) evdev: TWL4030 Keypad: Configuring as keyboard
[    48.848] (**) Option "config_info" "udev:/sys/devices/platform/68000000.ocp/48070000.i2c/i2c-1/1-00
48/48070000.i2c:twl@48:keypad/input/input2/event1"
[    48.848] (II) XINPUT: Adding extended input device "TWL4030 Keypad" (type: KEYBOARD, id 6)
[    48.848] (**) Option "xkb_rules" "evdev"
[    48.849] (**) Option "xkb_model" "nokiarx51"
[    48.849] (**) Option "xkb_layout" "us"
[    50.249] (II) config/udev: Adding input device twl4030_pwrbutton (/dev/input/event0)
[    50.250] (**) twl4030_pwrbutton: Applying InputClass "evdev keyboard catchall"
[    50.267] (II) Using input driver 'evdev' for 'twl4030_pwrbutton'
```

It seems to come from /etc/default/keyboard:

```
pavel@n900:~$ cat /etc/default/keyboard
# KEYBOARD CONFIGURATION FILE

# Consult the keyboard(5) manual page.

XKBMODEL="nokiarx51"
XKBLAYOUT="us"
XKBVARIANT=""
XKBOPTIONS=""

BACKSPACE="guess"

```

**Comment by pavelmachek** [comment]
This is how it works in Debian: https://wiki.debian.org/Keyboard

**Comment by pavelmachek** [comment]
This fixes it:

```
localhost:~$ cat /etc/X11/xorg.conf.d/40-xkb.conf 
Section "InputClass"
    Identifier "keyboard defaults"
    MatchIsKeyboard "on"

    Option "XkbModel" "nokiarx51"
    Option "XkbLayout" "us"
    Option "XKbOptions" ""
EndSection

```

**Comment by ollieparanoid** [comment]
Thanks for making a PR in #735.

**Comment by drebrez** [comment]
Fixed in #1177
